wolfssl/wolfcrypt/asn.h

```diff
@@ -2045,6 +2045,11 @@ WOLFSSL_API int wc_CheckCertSigPubKey(const byte* cert, word32 certSz,
                                       void* heap, const byte* pubKey,
                                       word32 pubKeySz, int pubKeyOID);
 #endif
+#if (defined(HAVE_ED25519) && defined(HAVE_ED25519_KEY_IMPORT) || \
+    (defined(HAVE_ED448) && defined(HAVE_ED448_KEY_IMPORT)))
+int wc_CertGetPubKey(const byte* cert, word32 certSz,
```

Comment:
Should have WOLFSSL_LOCAL, WOLFSSL_API, or WOLFSSL_ASN_API  in front.

Reply:
Fixed: WOLFSSL_LOCAL